typescript/web/src/components/export-button/export-modal/export-card.tsx

```diff
@@ -92,7 +97,16 @@ export const ExportCard = ({
           spacing={4}
           boxSizing="border-box"
         >
-          <Image src={logoSrc} w="16" flexGrow={0} flexShrink={0} />
+          {LogoIcon ? (
+            <LogoIcon
+              fontSize="64"
+              color={mode("black", "white")}
+              flexGrow={0}
+              flexShrink={0}
+            />
+          ) : (
+            <Image src={logoSrc} w="16" flexGrow={0} flexShrink={0} />
+          )}
```

Comment:
Can you move this block into an ExportIcon component so that we don't increare ExportCard complexity?